fractions/monitor/src/main/java/org/wildfly/swarm/monitor/runtime/HttpContexts.java

```diff
@@ -150,14 +151,13 @@ private void proxyRequests(HttpServerExchange exchange) {
 
                     int i = 0;
                     for (InVMResponse resp : responses) {
-                        if (200 == resp.getStatus()) {
-                            sb.append(resp.getPayload());
-                        } else if (503 == resp.getStatus()) {
-                            sb.append(resp.getPayload());
-                            failed = true;
-                        } else {
-                            throw new RuntimeException("Unexpected status code: " + resp.getStatus());
+
+                        sb.append(resp.getPayload());
+
+                        if (!failed) {
+                            failed = resp.getStatus() != 200;
```

Comment:
for what it's worth, I think any response between 200 and 299 is considered non-failed.  Maybe irrelevant given what this bit does, but a strict 200 check might be too strict.